tests/models/test_openai_responses.py

```diff
@@ -6320,3 +6324,2024 @@ class CityLocation(BaseModel):
+                        provider_name='openai',
+                    ),
+                    BuiltinMCPToolCallPart(
+                        tool_name='mcp_server:mcp_call_tool',
+                        args={
+                            'arguments': '{"minimal_output":true,"order":"desc","page":1,"perPage":100,"query":"user:pydantic"}'
```

Comment:
As mentioned above, I want to add the `server_id` and `tool_name` into these `args` again.